modules/web/web-api/src/main/java/com/enonic/xp/web/vhost/ResolveVirtualHostParams.java

```diff
@@ -0,0 +1,43 @@
+package com.enonic.xp.web.vhost;
+
+import javax.servlet.http.HttpServletRequest;
+
+import com.enonic.xp.annotation.PublicApi;
+
+@PublicApi
+public class ResolveVirtualHostParams
+{
+    private String serverName;
+
+    private String requestURI;
+
+    public ResolveVirtualHostParams()
+    {
+    }
+
+    public ResolveVirtualHostParams( final HttpServletRequest httpServletRequest )
```

Comment:
this us handy constructor, but better to avoid dependency on HttpServletRequest. So remove it